go.mod

```
@@ -0,0 +1,3 @@
+module github.com/tarantool/go-storage
+
+go 1.23
```

```suggestion
go 1.25
```
Let's use the latest version.

---

Nit, https://go.dev/doc/devel/release#policy , it should be 1.24.

---

Not quite understand why, but ok, it's changed to 1.24.